cross/ccrx.txt

```diff
@@ -0,0 +1,20 @@
+# This file assumes that path to the arm compiler toolchain is added
+# to the environment(PATH) variable, so that Meson can find
+# the armcc, armlink and armar while building.
```

Comment:
There's some copy-n-pasta from the arm toolchain here.

Reply:
Fixed up 👍